README.rst

```diff
@@ -142,6 +142,16 @@ You should now be able to browse to a ``Genes`` page and search for
 ``CDC20`` or ``ENSG00000117399``.
 
 
+External URL Favicons
+^^^^^^^^^^^^^^^^^^^^^
+
+Mapr can automatically convert external URLs in mapr categories to clickable favicons.
```

Comment:
While reviewing this change on IDR, I am realising there is an extra condition for this UI conversion which only seems to apply to keys defined in the mapr config.

See current production IDR (`prod88`)
<img width="1552" alt="Screenshot 2020-09-29 at 17 00 30" src="https://user-images.githubusercontent.com/1355463/94583463-6262bf80-0275-11eb-8ded-262dc81cf67b.png">

and `pilot-idr0094` with this PR included

<img width="1552" alt="Screenshot 2020-09-29 at 17 00 28" src="https://user-images.githubusercontent.com/1355463/94583430-5a0a8480-0275-11eb-878d-6375648d3feb.png">

Overall, this means the impact of this change should already be limited but that's certainly something worth adding to the README to clarify the expectations.

Reply:
That's what `in mapr categories` is meant to suggest. I was trying to come up with some reasonably concise wording.

Reply:
Ah thanks, I overlooked the wording but I am also trying to clarify the expectation for my own understanding ;)

In the example above, the `Compound Broad Identifer URL` key is in the `Compound Supplementary` mapr category which  namespace is defined in https://github.com/IDR/deployment/blob/78382a5adf4e69957a1e0aed8b660d3a230377f2/ansible/group_vars/omero-hosts.yml#L374-L380. However the namespace does not appear to be sufficient, so I assume the parent key must be explicitly defined in the app configuration as in https://github.com/IDR/deployment/blob/78382a5adf4e69957a1e0aed8b660d3a230377f2/ansible/group_vars/omero-hosts.yml#L364-L373 ?

Reply:
Yes, it has to be a category in the mapr configuration
https://github.com/ome/omero-mapr/blob/89f9c87fba77be8f0e9b7a532400a0c6d57902a7/omero_mapr/templates/webclient/data/includes/right_plugin.general.js.html#L71

Reply:
@manics I think you need to NOT be concise. This is really hard to understand what you need to put into the mapr config and what needs to match with what. I guess it depends a lot whether we want anyone external to use this feature, but even if it's just for within the team, the more info the better.

Reply:
Can you suggest some wording? Everything I can think of ends up being a jumble of buzzwords that doesn't really explain things either.

Reply:
I don't yet understand what's required to make this work, so can't suggest wording currently.
Do I need to start by installing and configuring redis?
What's the minimum mapr config and key-value pairs I need to use?
I'll try and see how far I can get...

Reply:
OK, I pushed some suggestions to https://github.com/will-moore/omero-mapr/commits/mapr-url-find

Reply:
Thanks! I've pulled all your commits.